cmd/root.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/executable"
+	"github.com/spf13/cobra"
+)
+
+var rpc string
+var proposal string
+var executableProposal *executable.ExecutableMCMSProposal
+var chainSelector string
+var pk string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "mcms",
+	Short: "Tools for on-chain interactions with the MCMS ",
+	Long:  ``,
+}
+
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
```

Comment:
Should we add unit test for those command? https://gianarb.it/blog/golang-mockmania-cli-command-with-cobra